deploy/strongswan.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
Yes, we must use `sh`.
You can check the original file on github, it's `sh`, not `bash`.
When installing on your system, we detect if `bash` is found, if found we will change the shebang to us `bash`,  since bash has a better performance in some of the operations.